data/human/human missions.txt

```diff
@@ -5428,3 +5428,214 @@ mission "Deal Change 2"
+				decline
+
+	on complete
+		payment 58000
+		conversation
+			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. They guide both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a repulsor taxi while he arranges for the <payment> to be sent to you.`
+			`	"Well, I must go now to the University of New Sydney. They and my daughter are expecting me at their Health Sciences quarter."`
+			choice
+				`	"Goodbye."`
+					goto bye
+				`	"I hope your daughter has a safe recovery."`
+					goto recovery
+				`	"Everything is going to be alright."`
+					goto recovery
+				`	"Could I come with you?"`
+					goto comealong
+			label bye
+			`	"Bye, now. I wish you greater luck in your travels and your sights. Take it from me, a man who cannot walk or see by himself, without aid; there is no greater sight than to see." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going towards University district on the far side of town.`
+				decline
+
+			label recovery
+			`	"Yes, thank you so much. I hope so too, of course; actually, it's a little foolish saying that, of course I do. But now I have to go. Again, thank you for making this journey possible."`
+			choice
+				`	"Goodbye."`
+					goto bye
+				`	"Could I come with you?"`
+					goto comealong
+			
+			label comealong
+			`	Douglas looks taken aback for a moment, but says, "Well, I suppose you can come along. I'm not sure if the university would let you in, though. But I think Anita would like to meet you, or at least say hello." You get into the taxi with him, and the taxi starts towards New Sydney's University district.`
+
+			# The player can do three actions before arriving at the hospital.
+			label taxi
+			branch questions hospital
+				"Blind Man from Martini: taxi questions count" != 3
+			label questions
+			choice
+				`	"How did you become blind?"`
+					goto blind
+					to display
+						not "Blind Man from Martini: label blind"
+
+				`	"How did becoming blind change your life?"`
+					goto blind-1
+					to display
+						has "Blind Man from Martini: label blind"
+						not "Blind Man from Martini: label blind-1"
+
+				`	"How do you feel about the atomic bombings?"`
+					goto atomic-bombings
+					to display
+						has "Blind Man from Martini: label blind"
+						not "Blind Man from Martini: label atomic-bombings"
+
+				`	"Was your daughter caught in the atomic bombings?"`
+					goto daughter
+					to display
+						or
+							has "Blind Man from Martini: label blind"
+							has "Blind Man from Martini: label illness"
+						not "Blind Man from Martini: label daughter"
```

Comment:
If you first ask him about his daughter as opposed to his blindness, your further option will include asking if she was caught in the nukes; but you don't necessarily know if _he_ was, either. It seems kind of an "off" question, if neither of you had mentioned nukes yet. 

I believe it should be further locked behind asking him about his blindness, going something like this: `"What happened to your daughter?"` -> `"How did you become blind?"`[only option] -> `"Was your daughter caught in the atomic bombings?"`, if you ask him about her first. ]

Also: 
```suggestion
						has "Blind Man from Martini: label blind"
						has "Blind Man from Martini: label illness"
						not "Blind Man from Martini: label daughter"
```
Those 2 above first lines seem to have an extra tab.

Reply:
The option is only meant to display if any of you mention nukes, which is why there's an "or" there and the lines are indented more. Maybe it doesn't work properly?

The option should not display if you've asked him already about it (label daughter)

Reply:
I'd check again and with the changes, but sadly trying to download the updated artefact doesn't work at the moment, and neither does ESL2 (though the latter wasn't working earlier today, either). Not sure why, but i click to download and GitHub crashes. Weird! 

I'll get back to you once it works again, i suppose! And about the commits, great changes, and about the name, you're very much welcome :D

Reply:
Alright, managed to redownload. And nope, it seems to not work properly! Here:
![image](https://user-images.githubusercontent.com/81255699/222838678-6abf8645-acbb-452d-9631-09370d4b63d9.png)

Secondly, when testing new paths i realised that this order was possible: 
![image](https://user-images.githubusercontent.com/81255699/222838328-0a21c731-1383-4d0b-8eee-416a8c1050df.png)
I believe that it would flow best if `what happened to your daughter` was locked out once you asked about his blindness, and only available again once you ask if she was caught in the bombing. Alternatively, you could lock out `Was your daughter caught in the atomic bombings` once he effectively answers she was not in the other question about her. Otherwise, it just seems a little off, in my opinion. It feels like the convo is the wrong way around if you "invert" that order, as he answers that her problems don't stem from the atomic bombs but from something on another planet! Either of those solutions would fix that.

Reply:
Mmm, I'll switch the two `daughter` labels around. Thanks.